.github/workflows/build-bindings-windows.yml

```diff
@@ -59,12 +59,31 @@ jobs:
     - name: Build sdk-bindings
       working-directory: libs/sdk-bindings
       run: cargo build --release --target ${{ matrix.target }}
-    
+
+    - name: Copy VC redistributables to release directory for Windows
+      if: matrix.target == 'x86_64-pc-windows-msvc'
+      working-directory: libs/target/${{ matrix.target }}/release
+      run: |
+        Copy-Item (vswhere -latest -find 'VC\Redist\MSVC\*\x64\*\msvcp140.dll') .
+        Copy-Item (vswhere -latest -find 'VC\Redist\MSVC\*\x64\*\vcruntime140.dll') .
+        Copy-Item (vswhere -latest -find 'VC\Redist\MSVC\*\x64\*\vcruntime140_1.dll') .
+
+    - name: Copy VC redistributables to release directory for Windows
+      if: matrix.target == 'i686-pc-windows-msvc'
+      working-directory: libs/target/${{ matrix.target }}/release
+      run: |
+        Copy-Item (vswhere -latest -find 'VC\Redist\MSVC\*\x86\*\msvcp140.dll') .
+        Copy-Item (vswhere -latest -find 'VC\Redist\MSVC\*\x86\*\vcruntime140.dll') .
+
     - name: Archive release
       uses: actions/upload-artifact@v3
       with:
         name: sdk-bindings-${{ matrix.target }}
-        path: libs/target/${{ matrix.target }}/release/breez_sdk_bindings.dll
+        path: |
+          libs/target/${{ matrix.target }}/release/breez_sdk_bindings.dll
+          libs/target/${{ matrix.target }}/release/msvcp140.dll
+          libs/target/${{ matrix.target }}/release/vcruntime140.dll
+          libs/target/${{ matrix.target }}/release/vcruntime140_1.dll
```

Comment:
Note: these will also be included in the C# package now. Question is whether that's problematic or not.
For golang I guess they're not included, because they're not git added before tagging the go release.

Reply:
I'll look into this

Reply:
Changed it so the redistributable DLLs are only included when building the windows wheels